.github/workflows/php-tests-wp-nightly-php-7-4.yml

```diff
@@ -75,4 +75,4 @@ jobs:
       - name: Set up PHP test data
         run: tests/bin/install-wp-tests.sh ${MYSQL_DATABASE} ${MYSQL_USER} ${MYSQL_PASSWORD} ${DB_HOST}:${DB_PORT} ${WP_VERSION}
       - name: Run Unit Tests
-        run: composer test
+        run: composer test -- --verbose
```

Comment:
Not opposed, but curious why you changed this throughout.

Reply:
Mostly because it adds information about the runtime to the run:

```
Runtime:       PHP 8.0.14
Configuration: /home/runner/work/site-kit-wp/site-kit-wp/phpunit.xml.dist
```

It also adds things like the names of tests that were incomplete or have been skipped.

If we really wanted to make our output more verbose (like Jest) we could use `--testdox`, but `--verbose` doesn't really add that much. I mainly wanted to expose the specific version of PHP used.